docker/lnd/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.8
+FROM golang:1.9
```

Comment:
Why not 1.10?

Reply:
Yeah, looks like this builds fine with 1.10 so may as well use the latest.
```
Building lnd
Step 1/11 : FROM golang:1.10
 ---> 1c1309ff8e0d
...
Step 9/11 : RUN go install . ./cmd/...
 ---> Running in 19fa779471b5
Removing intermediate container 19fa779471b5
 ---> e272e3567653
...
Successfully built cbb6e9e99062
Successfully tagged lnd:latest
```
Should I squash into 1 commit, or leave it 2 since review has technically started?